cpp/ps0000/p0.cpp

```diff
@@ -1,9 +1,10 @@
-#include <iostream>
+#include <iostream> // 2/2

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`